components/cam/cime_config/config_compsets.xml

```diff
@@ -371,18 +371,28 @@
     <alias>FSP1V1</alias>
     <lname>2000_CAM5%SP1V1_CLM45%SPBC_CICE%PRES_DOCN%DOM_SROF_SGLC_SWAV</lname>
   </compset>
+  
+  <compset>
+    <alias>FSP1V1-MAML</alias>
+    <lname>2000_CAM5%SP1V1-MAML_CLM45%SPBC_CICE%PRES_DOCN%DOM_SROF_SGLC_SWAV</lname>
```

Comment:
A note about SPBC - I think the "BC" part is why we've had some confusion about the "bc_dep_to_snow_updates" flag. I think we can omit this flag if we take off the "BC" for 1-mom runs since we are using prescribed aerosols. We can do this in another PR though since I'm sure it will be non-BFB.